### PR TITLE
fix(#7791): tc form shouldn't submit bounds changes on dismiss

### LINKED ISSUE #7791: [Time Conductor] In fixed time mode, dismissing bounds changes actually submits them
#### Summary

When making changes to time conductor bounds in fixed mode, dismissing the changes via the 'X' button will submit them anyway. 

Discovered due to [failing tests in openmct-yamcs](https://github.com/akhenry/openmct-yamcs/actions/runs/10080695502/job/27871032510?pr=465#step:13:219) 

#### Expected vs Current Behavior

Clicking 'Dismiss' should not submit any changes

#### Steps to Reproduce

1. Switch to fixed time mode in the global time conductor
2. Change the start year to something else (e.g. 2024 -> 2023)
3. Dismiss the changes by clicking the 'X'
4. Check the time conductor and see that the change has actually been submitted

#### Environment

* Open MCT Version:  4.0.0-next
* Deployment Type:  local, openmct-yamcs
* OS: any
* Browser: chrome

#### Impact Check List

- [ ] Data loss or misrepresented data?
- [x] Regression? Did this used to work or has it always been broken?
- [ ] Is there a workaround available?
- [x] Does this impact a critical component?
- [ ] Is this just a visual bug with no functional impact?
- [ ] Does this block the execution of e2e tests?
- [ ] Does this have an impact on Performance?

#### Additional Information

https://github.com/user-attachments/assets/b5541ff3-3a5c-47eb-bac1-6f0a1f48de50

### PR DESCRIPTION
**Pull request by ozyx**
Closes #7791 

### Describe your changes:

Reverts some changes that caused the `dismiss` action on the fixed mode time conductor popup to submit changes. Does some "safe" refactoring of the component to make it slightly more maintainable.

* Updates `setTimeConductorBounds` to be more robust
* Update usage of `setTimeConductorBounds` in existing tests
* Adds a regression test

### All Submissions:

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/nasa/openmct/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/nasa/openmct/pulls) for the same update/change?
* [ ] Is this a [notable change](../docs/src/process/release.md) that will require a special callout in the release notes? For example, will this break compatibility with existing APIs or projects that consume these plugins?

### Author Checklist

* [x] Changes address original issue?
* [x] Tests included and/or updated with changes?
* [x] Has this been smoke tested?
* [x] Have you associated this PR with a `type:` label? Note: this is not necessarily the same as the original issue.
* [x] Have you associated a milestone with this PR? Note: leave blank if unsure.
* [x] Testing instructions included in associated issue OR is this a dependency/testcase change?

### Reviewer Checklist

* [ ] Changes appear to address issue?
* [ ] Reviewer has tested changes by following the provided instructions?
* [ ] Changes appear not to be breaking changes?
* [ ] Appropriate automated tests included?
* [ ] Code style and in-line documentation are appropriate?

### CODE REVIEW

**Comment by unlikelyzero** [comment]
Let's make sure to add a test for this scenario with the native time conductor, but also an embedded independent time conductor in a display layout

**Comment by ozyx** [comment]
> Let's make sure to add a test for this scenario with the native time conductor, but also an embedded independent time conductor in a display layout

Working on it rn 👍

**Review by unlikelyzero — APPROVED** [review]
One small reminder to update the test framework test app action for the set time conductor bounds

File: e2e/tests/functional/plugins/imagery/exampleImagery.e2e.spec.js [hunk 1]
```diff
@@ -53,6 +53,9 @@ test.describe('Example Imagery Object', () => {
     // Verify that the created object is focused
     await expect(page.locator('.l-browse-bar__object-name')).toContainText(exampleImagery.name);
     await page.getByLabel('Focused Image Element').hover({ trial: true });
+
+    // Wait for image thumbnail auto-scroll to complete
+    await expect(page.getByLabel('Image Thumbnail from').last()).toBeInViewport();
```
Comment (ozyx) [inline e2e/tests/functional/plugins/imagery/exampleImagery.e2e.spec.js:58]: GOT YOU!!

gremlins5

Comment (ozyx) [reply e2e/tests/functional/plugins/imagery/exampleImagery.e2e.spec.js:58]: here lies `Can use reset button...` test flake (2022-2024) 🪦

**Review by unlikelyzero — APPROVED** [review]

**Pull request merged**